blackbox/docs/admin/cluster-management.rst

```diff
@@ -0,0 +1,101 @@
+.. _cluster_management:
+
+==================
+Cluster management
+==================
+
+With the ``crate-node`` CLI tool you can manage your CrateDB clusters by forming
+new clusters, repairing broken clusters, and by repairing broken nodes.
+
+The ``crate-node`` CLI tool
+---------------------------
+
+The ``crate-node`` CLI tool lets you:
+
+ * Unsafely bootstrap a cluster
+ * Detach a node from its cluster
+ * Change the role of a node (and may be able to recover some data after a
+   disaster)
+
+.. NOTE::
+
+  Cluster management operations on a node are only possible while the node is shut down.
+
+Synopsis
+~~~~~~~~
+
+.. code-block:: console
+
+   bin/crate-node repurpose|unsafe-bootstrap|detach-cluster
```

Comment:
where is this taken from? the syntax confuses me

Reply:
Here's the source: https://www.elastic.co/guide/en/elasticsearch/reference/current/node-tool.html